setup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+curl -sL https://install.raspap.com | bash
```

Comment:
This not working if container didn't have latest ssl certificates, you need to update it before (because the source image alehaa/debian-systemd:buster got an expired one at this time) or add -k args to ignore it. (Tell me if I'm wrong, but ssl didn't work for me)